terraform/variables.tf

```diff
@@ -73,3 +73,71 @@ variable "public_vm_count" {
 variable "sql-server-version" {
   default = "12.0"
 }
+
+
+# Azure mysql
```

Comment:
As above, it's fine to hardcode most/all of these straight into the terraform resource, unless they are being used in several places!